raster-foundry/raster-foundry
# Update Lodash and node-sass dependencies

## Overview

This allows us to side-step a CVE for Lodash and remove a post-install step for `node-sass`.

In addition:

- Remove empty description
- Add license to `package.json`
- Remove stray `yarn.lock`
- Switch to using official Node.js v6 image (includes `yarn`)

Fixes https://github.com/raster-foundry/raster-foundry/issues/3814

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] [Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~


~- [ ] Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
~- [ ] Any new SQL strings have tests~

## Notes

Inspect the patch with whitespace ignored to more clearly see the changes made to `package.json`.

## Testing Instructions

- Once on this branch, execute `update` so that the static bundle is cleared and rebuilt
- Afterwards, execute `server` to ensure that the frontend still renders without errors
- Interact with the application
